docs/faq/state-tree.md

```diff
@@ -14,8 +14,8 @@ resource's reducer.
 
 Yes, you can. The only requirement is that you don't change the structure of the
 state that you start out with: make sure that `resources` remains an Array,
-and that `meta` and `labels` remain Objects. If you stick with that, then you
-shouldn't run into any issues.
+and that `meta`, `lists`, and `labels` remain Objects. If you stick with that, then
+you shouldn't run into any issues.
```

Comment:
nit: `If you stick with that you shouldn't run into any issues`

Reply:
I’ll update! Thanks :v: